plugin/testdata/plugins.conf

```diff
@@ -0,0 +1,5 @@
+[PLUGINS]
+    Path /fluent-bit/etc/go-test-input-plugin.so
```

Comment:
I still think the plugin name and description should be placed here and not in the plugin code itself.
If you share a plugin by just downloading an `.so` file there is no way of knowing the plugin name 🤔

Reply:
+1 we should extend this optionally for allowing a name to be passed. lets keep it as is for now.

Reply:
-1 I don’t think we should provide plugin name and description extensibility in config file. IMO, I don’t want to introduce this complexity but this is only my two cents.

Reply:
-1. I agree with Hiroshi, there is no reason we should force people to guess what the internal name of a plug-in is when trying to load it.

The only use I can think of is to override a plugins name when loading it. That might be useful but a bit out of scope for the moment.

Reply:
I see it the opposite. You have to guess it right now.

I propose that the plugin itself has no name or description at all in its code.

When you load it you have to name it.

Reply:
> -1. I agree with Hiroshi, there is no reason we should force people to guess what the internal name of a plug-in is when trying to load it.
> 
> The only use I can think of is to override a plugins name when loading it. That might be useful but a bit out of scope for the moment.

I agree with this approach. Lets keep it as defined by the plugin for the time being.